library/src/template-proto.awk

```diff
@@ -0,0 +1,14 @@
+# multi-line
```

Comment:
`s/awk/perl/`

Reply:
s/awk/gawk but it looks like will have to s/gawk/python

Reply:
Use whatever tool is best for the job. TIMTOWDI!!!